### DISCUSSION TITLE
GH Actions not triggering for Dependabot PRs

### DISCUSSION THREAD

**Discussion by gerryfletch** [community/community#25445] [Actions]
<p>My <code>dependabot.yml</code> is specified as so:</p>
<pre><code class="lang-auto">version: 2
updates:
  # Enable version updates for npm
  - package-ecosystem: 'npm'
    # Look for `package.json` and `lock` files in the `root` directory
    directory: '/'
    # Check the npm registry for updates every day (weekdays)
    schedule:
      interval: 'daily'
</code></pre>
<p>And I register my GitHub actions tests for all branches except master:</p>
<pre><code class="lang-auto">name: Test

on:
  push:
    branches:
      - '*'
      - '!master'

jobs:
  ....
</code></pre>
<p>The tests run for all PRs I create. Dependabot uses the branch style <code>dependabot/npm_and_yarn/package_org/package-version</code> and merges into master, but the GitHub action is never triggered. Example: <a href="https://github.com/two-app/two-app/pull/166" rel="noopener nofollow ugc">https://github.com/two-app/two-app/pull/166</a></p>
<p>I can’t find other threads with the same issues, but have seen some users running their CI for dependabot PRs. Any ideas?</p>

**Comment by Simran-B** [comment, marked as answer]
<pre><code class="lang-auto">branches:
  - '*'
</code></pre>
<p><code>*</code> only matches everything up to the first <code>/</code>. It should match the Dependabot branch names if you make it <code>**</code>.</p>
<p><a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet" rel="noopener nofollow ugc">Filter pattern cheat sheet</a></p>
